PyInstaller/loader/pyiboot01_bootstrap.py

```diff
@@ -111,6 +111,23 @@ def flush(*args):
 if sys.warnoptions:
     import warnings
 
+try:
+    import ctypes
+    import os
+    from ctypes import LibraryLoader, DEFAULT_MODE
+
+
+    class PyInstaller_CDLL(ctypes.CDLL):
+        def __init__(self, name, mode=DEFAULT_MODE, handle=None, use_errno=False, use_last_error=False):
+            frozen_name = os.path.join(sys._MEIPASS, os.path.basename(name))
+            if os.path.exists(frozen_name):
+                name = frozen_name
+            super(PyInstaller_CDLL, self).__init__(name, mode, handle, use_errno, use_last_error)
+
+    ctypes.CDLL = PyInstaller_CDLL
+    ctypes.cdll = LibraryLoader(PyInstaller_CDLL)
+except ImportError:
+    pass
 
```

Comment:
This is something I wanted implement one day. Thanks! Please add comment what this 'try' block is doing (looking first for dlls in MEIPASS).